dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.auditWebProbes1.0.jakarta.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName =io.openliberty.auditWebProbes1.0.jakarta
+visibility = private
+IBM-Install-Policy: when-satisfied
+
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.audit1.0.jakarta))"
```

Comment:
This filter doesn't make any sense.  Just move the bundles from this auto feature to the audit1.0.jakarta auto feature.